ipa-core/Cargo.toml

```diff
@@ -193,6 +193,11 @@ name = "test_mpc"
 required-features = ["cli", "test-fixture", "web-app", "weak-field"]
 bench = false
 
+[[bin]]
+name = "crypto_util"
+required-features = ["cli", "test-fixture", "web-app"]
```

Comment:
```suggestion
required-features = ["cli", "test-fixture"]
```

Reply:
I don't think you need `web-app` feature

Reply:
probably `test-fixture` is redundant too

Reply:
just tested and removing `web-app` is fine. `test-fixture` is used here:

```
error[E0433]: failed to resolve: could not find `test_fixture` in the crate root
  --> ipa-core/src/cli/crypto.rs:23:5
   |
23 |     test_fixture::{ipa::TestRawDataRecord, Reconstruct},
   |     ^^^^^^^^^^^^ could not find `test_fixture` in the crate root
   ```

Reply:
oh, looks like I'm wrong. I use `cli::playbook` , and that relies on `web-app`.

Reply:
heh, yea - these dependencies are not really `web-app` specific, but it is not the purpose of this PR to fix them.